google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/Exceptions.java

```diff
@@ -259,6 +259,19 @@ public AppendSerializationError(
     }
   }
 
+  /** This exception is thrown from proto converter to wrap the row index to error mapping. */
+  static class RowIndexToErrorException extends IllegalArgumentException {
+    Map<Integer, String> rowIndexToErrorMessage;
+
+    public RowIndexToErrorException(Map<Integer, String> rowIndexToErrorMessage) {
+      this.rowIndexToErrorMessage = rowIndexToErrorMessage;
+    }
+
+    public String getMessage() {
```

Comment:
Put a comment that this getMessage shouldn't be returned to users directly, instead they should always use rowIndexToErrorMessage.

Reply:
Done